docs/source/pyplots/calc_sphere.py

```diff
@@ -1,11 +1,16 @@
 import holopy as hp
 from holopy.scattering import calc_holo, Sphere
+from holopy.scattering.theory import Mie
+
+sphere = Sphere(n=1.59, r=0.5, center=(4, 4, 5))
 
-sphere = Sphere(n = 1.59, r = .5, center = (4, 4, 5))
 medium_index = 1.33
 illum_wavelen = 0.660
-illum_polarization = (1,0)
-detector = hp.detector_grid(shape = 100, spacing = .1)
+illum_polarization = (1, 0)
+detector = hp.detector_grid(shape=100, spacing=0.1)
+
+theory = Mie()
```

Comment:
Most users of HoloPy don't need to worry about specifying theories, since the automatic determination based on Scatterer type should be sufficient. Since this is the definitive example of a scattering calculation, I don't think we should clutter it with a specification of a scattering theory.

Reply:
Let's talk about this tomorrow!